distribution/ddf-common/src/main/resources/etc/templates/example.ManagedService.config

```diff
@@ -1,22 +1,42 @@
-property.string="string"
-property.boolean.true=B"true"
-property.boolean.false=B"false"
-property.int=I"10"
-property.long=L"100"
-// Floats and doubles are supported using the IEEE 754 format. See the Documentation for more details.
-property.float=F"0x41280000"
-property.double=D"0x42c83f2e"
+property.string = "string"
+property.boolean.true = B"true"
+property.boolean.false = B"false"
+property.int = I"10"
+property.long = L"100"
+# Floats and doubles are supported using the IEEE 754 format. See the Documentation for more details.
```

Comment:
❓ Do we need to update the documentation where we explain how to convert from a float or double to the IEEE 754 format that needs to be used in the configuration files?

Reply:
I added links to the java documentation to provide some more details about the conversions.